src/modules/Dropdown/Dropdown.js

```diff
@@ -85,6 +85,9 @@ export default class Dropdown extends Component {
     /** Whether or not the menu should close when the dropdown is blurred. */
     closeOnBlur: PropTypes.bool,
 
+    /** Whether or not the menu should close when a value is selected from the dropdown */
```

Comment:
Since the default is hidden from the user (logic based in the method) perhaps we should note the default here.  Let's add the period to the end of this sentence while we're at it.